index.js

```diff
@@ -8,6 +8,8 @@ var globparent = require('glob-parent');
 var isglob = require('is-glob');
 var arrify = require('arrify');
 var isAbsolute = require('path-is-absolute');
+var flatten = require('lodash.flatten');
+var isString = require('is-string');
```

Comment:
do we really need a separate module for `.filter(item => typeof item === 'string')`?